wdl/MainVcfQc.wdl

```diff
@@ -0,0 +1,843 @@
+version 1.0
+
+# Note: this WDL has been customized specifically for the gnomAD-SV v3 callset
+# Some components of this WDL will not be generalizable for most cohorts
+
+import "ShardedQcCollection.wdl" as ShardedQcCollection
+import "CollectQcPerSample.wdl" as CollectQcPerSample
+import "ShardedCohortBenchmarking.wdl" as CohortExternalBenchmark
+import "PerSampleExternalBenchmark.wdl" as PerSampleExternalBenchmark
+import "TasksMakeCohortVcf.wdl" as MiniTasks
+import "Utils.wdl" as Utils
+
+# Main workflow to perform comprehensive quality control (QC) on
+# an SV VCF output by GATK-SV
+workflow MainVcfQc {
+  input {
+    Array[File] vcfs
+    Array[File] vcf_idxs
+    Boolean vcf_format_has_cn = true
+    File? ped_file
```

Comment:
Make sure to update the inputs to MainVcfQc in MakeCohortVcf (& anywhere else that MainVcfQc is invoked), validate all the WDLs and JSONs with womtool, and test MakeCohortVcf